packages/p2p/src/components/my-profile/payment-methods/payment-method-card/payment-method-card.jsx

```diff
@@ -102,11 +104,18 @@ const PaymentMethodCard = ({
                 )}
             </div>
             <div className='payment-method-card__body'>
-                <Text color='prominent' size={large ? 'xs' : 'xxs'}>
-                    {method === 'BankTransfer' || method === 'Other'
-                        ? payment_method?.display_name
-                        : payment_account_name}
-                </Text>
+                {show_payment_method_name && (
+                    <Text color='prominent' size={large ? 'xs' : 'xxs'}>
+                        {method === 'BankTransfer' || method === 'Other'
+                            ? payment_method?.display_name
+                            : payment_account_name}
+                    </Text>
+                )}
+                {!show_payment_method_name && method !== 'BankTransfer' && method !== 'Other' && (
+                    <Text color='prominent' size={large ? 'xs' : 'xxs'}>
+                        {payment_account_name}
+                    </Text>
+                )}
```

Comment:
Why're we adding the prop `show_payment_method_name` when we already have the condition on line 109 that shows `payment_account_name` if the method isn't bank transfer or other.

Reply:
In different pages the payment method names for types such as `Bank Transfer` or `Other` are shown and not shown according to the design, only e-wallet payment method names are shown in every page, the prop `show_payment_method_name` is used for controlling if the type `Bank Transfer` or `Other` should be shown in that particular page, for example:
*My Profile Payment Methods section, payment method names for types Bank Transfer and Other is not shown except for e-wallets according to design requirements*
![Screenshot 2022-05-10 at 11 47 01 AM](https://user-images.githubusercontent.com/103016120/167539420-25f0fd2d-6665-4bd5-97c8-50bea5459e47.png)
*Create Sell Ad, payment method names for types Bank Transfer, Other and e-wallet is shown according to design requirements**
![Screenshot 2022-05-10 at 11 46 38 AM](https://user-images.githubusercontent.com/103016120/167539269-41213cb4-c516-490d-97e1-f91474380274.png)
*Create Sell Order, payment method names for types Bank Transfer, Other and e-wallet is shown according to design requirements***
![Screenshot 2022-05-10 at 11 46 08 AM](https://user-images.githubusercontent.com/103016120/167539284-9cbe2920-7a98-4417-a6d9-d9d0a817f735.png)

Reply:
We can still add `show_payment_method_name` as part of the existing condition where we're checking for method